scripts/build.js

```diff
@@ -171,17 +171,17 @@ build.overlay_image = function(data) {
 	{
 		html = lychee.html`
 					<div id="image_overlay">
-						<h1> ${ data.title }</h1>
-						<p>${ data.description }</p>
+						<h1>$${ data.title }</h1>
+						<p>$${ data.description }</p>
 					</div>
 				`;
 	}
-	else if (type && type==='date' && data.date !== '')
+	else if (type && type==='takedate' && data.date !== '')
 	{
 		html = lychee.html`
 			<div id="image_overlay">
-				<h1>${ data.title }</h1>
-				<p>${ data.takedate }</p>
+				<h1>$${ data.title }</h1>
+				<p>$${ data.takedate }</p>
```

Comment:
the $ is not really needed here (for takedate) as the value is sanitized server side.